### DISCUSSION TITLE
docker部署halo自己挂掉是什么原因？

### DISCUSSION THREAD

**Discussion by llody55** [halo-dev/halo#4770] [Q&A]
### What is version of Halo has the issue?

2.5.2

### What database are you using?

H2

### What is the project operation method?

Docker

### Your site address.

https://www.llody.top/

### What happened?

我1核2G的主机，部署今天启动后过会儿会自己挂掉，报错信息没看懂呢
![image](https://github.com/halo-dev/halo/assets/9710645/f3c9eafe-2210-4a6d-bf72-f1512a68b338)

**Comment by JohnNiang** [comment]
可能是因为内存不足导致被 kill。建议先升级到最新版后重试。

**Comment by llody55** [comment]
我换了一个内存够的主机还是这样呢，运行不起来
![image](https://github.com/halo-dev/halo/assets/9710645/118eb8f2-3d31-41b7-bccd-f36359fc065f)

**Comment by JohnNiang** [comment]
似乎是 Docker 版本太低的原因，建议升级到 Docker 至最新后重试。

**Comment by ruibaby** [comment]
> 我换了一个内存够的主机还是这样呢，运行不起来 ![image](https://user-images.githubusercontent.com/9710645/277584739-118eb8f2-3d31-41b7-bccd-f36359fc065f.png)

https://github.com/halo-dev/halo/issues/2503

/kind support
